Experimental PH-TESS notebook

eatyourgreens opened the pull request:
A modification of the 1D sonification notebook. It replaces the generated 1D data with a couple of simulated TESS light curves from Planet Hunters TESS.

This is a notebook that I've been using to play with STRAUSS and learn how it works.

james-trayford commented:
Great to see these TESS examples! Interesting to deal with a combination of longer term trends but also some dramatic spikes, there may pertinent question around how to handle these, depending on the properties of interest.  Generally some of the existing approaches seem to work well I think. Also good to download the data in-notebook too, to keep the `data/` folder light.

May be good to name it `TESSlightcurves.ipynb` or similar to clarify (realising `Sonifying` is redundant in these example names, I should also change `SonifyingData1D.ipynb`).

I'm thinking it would be good to merge a TESS example once this has diverged more from `SonifyingData1D.ipynb` - particularly if this data set brings up particular issues or points of interest. Else we could integrate some TESS examples into the existing notebook.

eatyourgreens commented:
>May be good to name it TESSlightcurves.ipynb or similar to clarify (realising Sonifying is redundant in these example names, I should also change SonifyingData1D.ipynb).

Good idea! I've renamed the notebook.

>I'm thinking it would be good to merge a TESS example once this has diverged more from SonifyingData1D.ipynb - particularly if this data set brings up particular issues or points of interest. Else we could integrate some TESS examples into the existing notebook.

I've made a couple of changes now, and rebased. Sonifications are 20s long, not 15, and I've had a go at inverting the pitch mapping example. With the first subject (the default), you should hear 16 'chirps', corresponding to the 16 downward spikes in the light curve.

I haven't touched any of the other examples yet. I'm still playing around, figuring out how the mapping parameters work.

eatyourgreens commented on examples/TESSLightCurves.ipynb:
```diff
@@ -0,0 +1,275 @@
+   "source": [
+    "**Set up some universal sonification parameters and classes for the examples below**\n",
+    "\n",
+    "For all examples we use the `Synthesizer` generator to create a 30 second, mono sonification."
```
I think this line is a holdover from an earlier notebook. Anyway, the examples here are 20 seconds and stereo (panning from left to right as we move along the light curve.)

james-trayford commented on examples/TESSLightCurves.ipynb:
```diff
@@ -0,0 +1,279 @@
+    "        'time_evo':x,\n",
+    "        'azimuth':(x*0.5+0.25) % 1,\n",
+    "        'polar':0.5,\n",
+    "        'pitch_shift':(y*10+1)**-0.7}\n",
```
The `**-0.7` here means that higher flux => lower pitch - could just map `'pitch_shift':y` here instead to get high pitch => high flux?
